src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+package duke;
+
+/**
+ * TO DO:
+ * Update EXPECTED.TXT
+ */
```

Comment:
Please consider writing notes to yourself in a separate document and not in the code directly. Read [this](https://nus-cs2113-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#write-to-the-reader) part from the textbook to find out more.